packages/rsocket-flowable/src/Invariant.js

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
I believe this is change is a bit controversial because it introduces duplication between `rsocket-core` and `rsocket-flowable` even though it's small. To resolve the issue, we could either 
* Introduce a new package with similar helpers (e.g. `rsocket-util`)
* Make `rsocket-flowable` export `invariant` - everything depends on `flowable` already. 
* Accept the duplication and do nothing. 

WDYT?

Reply:
I would advise against the second option as it would then create a contractual obligation with consumers of `rsocket-flowable` for `Invariant` to be exposed by the package.

Personally, I think accepting the duplication could be sufficient as an intermediate step, however, we may also be able to explore something like https://www.npmjs.com/package/invariant to provide the same functionality.